torch/_export/serde/schema.py

```diff
@@ -7,7 +7,7 @@
 
 
 # NOTE: Please update this value if any modifications are made to the schema
-SCHEMA_VERSION = 1
+SCHEMA_VERSION = 2
```

Comment:
not blocking: should we add comment here describing what is changed? In general, whenever there is the version update, there should be some sort of doc or tracker?

Reply:
ooh. I'm not sure what should be done during a version bump. I can add it to this PR and make sure it's in the release notes?

Reply:
What if we paste it in the code as comment as well?